companies/serializers.py

```
@@ -263,3 +264,41 @@ def get_logo(self, obj):
+
+#sankey seiralizers
+class SankeySerializer(serializers.ModelSerializer):
+    company_name = serializers.CharField(source='company.company_name', read_only=True)
+    total_revenue = serializers.SerializerMethodField()
+    segments = serializers.SerializerMethodField()
+    expenses = serializers.SerializerMethodField()
+    fiscal_year = serializers.CharField()
+
+    class Meta:
+        model = SankeyData
+        fields = ['company_name', 'fiscal_year', 'total_revenue', 'is_loss', 'segments', 'expenses']
```

Ruff (0.14.13)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)